### PR TITLE
Add Ogmo Editor 3 map parser

### PR DESCRIPTION
**Pull request by yberreby**
This is _very_ preliminary:
* Outputs data to stdout (can be pretty-printed by piping to `xxd -b`)
* Reads input at compile time
* Doesn't sort by `clutter_level` yet

Not ready for review, even less for merging.

### CODE REVIEW

**Comment by yberreby** [comment]
@ChatPion Please have a look

File: ogmo/ogmo-parser/src/main.rs [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+
+    for entity in &ent_layer.entities {
+        let (i, j) = (entity.x / 8, entity.y / 8);
+
+        // Printing to stderr allows use to reserve stdout for the binary output.
+        eprintln!(
+            "Found entity with id {} at coords ({}, {})",
+            entity.values.id, i, j
+        );
+        let tile_idx = i * 8 + j;
```
Comment (ChatPion) [inline ogmo/ogmo-parser/src/main.rs:39]: need to swap i and j

Comment (yberreby) [reply ogmo/ogmo-parser/src/main.rs]: Done

**Comment by yberreby** [comment]
By the way, I don't think rebasing and merging would be appropriate here, the commit history is a total mess. So let's just squash it.

**Comment by ChatPion** [comment]
Could you approve #39 ?

**Pull request merged**